initrd/bin/network-init-recovery

```diff
@@ -0,0 +1,29 @@
+#!/bin/ash
+
+# bring up the ethernet; maybe should do DHCP?
+ifconfig lo 127.0.0.1
+
+if [ -f /lib/modules/e1000.ko ]; then
+	insmod /lib/modules/e1000.ko
+fi
+
+if [ -f /lib/modules/e1000e.ko ]; then
+	insmod /lib/modules/e1000e.ko
+fi
```

Comment:
The modules are device specific, right? Should that depend on some configuration?

Reply:
They are device specific.  I could use the variables in `module/linux` that control the inclusion of the kernel modules (`CONFIG_LINUX_E1000`, `CONFIG_LINUX_E1000E`, `CONFIG_LINUX_IGB`, `CONFIG_LINUX_SFC`, `CONFIG_LINUX_MLX4`) and export them in all the board configs, but that's equivalent to testing for existence in the initrd.  @osresearch what do you think?